src/v2/Apps/Consign/Routes/SubmissionFlow/UploadPhotos/UploadPhotos.tsx

```diff
@@ -1,12 +1,42 @@
 import React, { FC } from "react"
-import { Box } from "@artsy/palette"
+import { Box, Text } from "@artsy/palette"
+import { Formik } from "formik"
 import { SubmissionStepper } from "v2/Apps/Consign/Components/SubmissionStepper"
+import {
+  UploadPhotosForm,
+  UploadPhotosFormModel,
+} from "./Components/UploadPhotosForm"
 
 export const UploadPhotos: FC = () => {
+  const handleSubmit = () => {}
+
   return (
-    <>
+    <Box>
       <SubmissionStepper currentStep="Upload Photos" />
-      <Box mt={50}>Upload Photos step</Box>
-    </>
+
+      <Text variant="lg">Upload photos of your artwork</Text>
+      <Text variant="sm" color="black60" mt={1}>
+        • For a faster valuation, please upload high-quality photos of the
```

Comment:
nitpick: Is it OK to use unicode characters in HTML, or should we use an HTML code (`&#183;`) for the bullet-point character instead?

Reply:
I don't think that it's a problem, but I can update it to `&#8226`(a little bit bigger than `&#183;`) just in case.

Reply:
Fixed